content/docs/started/workstation/getting-started-macos.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Kubeflow on MacOS"
+description = "Install Kubeflow on MacOS"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+        more quickly
+  TODO: Surface the mac specific instructions here .. for instance, installing
+        vagrant and virtualbox through brew / brew cask
+-->
+
+## Kubeflow Appliance
+
+A Kubeflow appliance is a virtual machine that has Kubeflow already installed. Once the
+necessary supporting software is installed no further installation steps are required.
+
+### MiniKF
+
+MiniKF is a predefined virtual machine that installs onto VirtualBox through Vagrant.
+The only following applications are required to use MiniKF:
+
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+- Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+
```

Comment:
same as above, can you restore the MiniKF description as it was on the Getting Started page?

> MiniKF is a fast and easy way to get started with Kubeflow. It installs with just two commands and then you are up for experimentation, and for running complete Kubeflow Pipelines. It runs on all major operating systems (macOS, Windows, Linux).

Reply:
same here, I thought I had selected the whole description, but GitHub's multi-line review doesn't seem to work as expected.